diff -Nrdup a/BUILD.gn b/BUILD.gn
--- a/BUILD.gn	2021-09-01 12:51:12.369601814 +0800
+++ b/BUILD.gn	2021-09-01 12:51:32.106000486 +0800
@@ -68,10 +68,6 @@ if (defined(ohos_lite)) {
         "-lm",
         "-lpthread",
       ]
-      deps += [
-        "//third_party/mksh",
-        "//third_party/toybox",
-      ]
     }
   }

diff -Nrdup a/src/init_service.c b/src/init_service.c
--- a/src/init_service.c	2021-09-01 12:51:12.369601814 +0800
+++ b/src/init_service.c	2021-09-01 12:51:32.106000486 +0800
@@ -15,7 +15,8 @@
 
 #include "init_service.h"
 
-#include <bits/ioctl.h>
+//#include <bits/ioctl.h>
+#include <asm-generic/ioctls.h>
 #include <errno.h>
 #include <fcntl.h>
 #include <signal.h>
